app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+import StaffCard from './StaffCard'
+
+const { Flex, Box } = require('grid-styled')
+
+const Grid = styled(Flex)`
+  flex-flow: row wrap;
+`
+const Card = styled(Box)`
+  padding-top: 20px;
+`
+
+const StaffGrid = ({ staff }) => (
+    <Grid>
+      {staff.map(person => (
+        <Card Box px={2} width={1 / 3}>
```

Comment:
Is the word `Box` here a typo? I'm not sure that it's doing anything.

I reckon you might want media queries on `Card`? Right now, on mobile, it looks like this:
![image](https://user-images.githubusercontent.com/15656538/47420649-d980a680-d776-11e8-8bc6-985697e9e08a.png)

I imagine that we'll want some kind of breakpoint, after which this becomes a 2X2 grid. What do you think?

Also, is there any particular reason why we have `px={2}` (i.e. 12px) here, but the vertical spacing between cards is 20px?